includes/data-port/models/class-sensei-data-port-lesson-schema.php

```diff
@@ -103,8 +103,7 @@ public function get_schema() {
 				'default' => false,
 			],
 			self::COLUMN_PASSMARK       => [
-				'type'      => 'int',
-				'validator' => [ $this, 'validate_passmark' ],
+				'type' => 'decimal',
```

Comment:
The restore method `\Sensei_Import_Model::restore_from_source_array` currently accepts `float` for this.

Reply:
Oops! Fixed here: https://github.com/Automattic/sensei/pull/3432/commits/9f75992a88a398423057984cac7a40bdcc3429a7

Thank you for your reviews, Jake! \o/